README.md

```diff
@@ -41,6 +41,10 @@ spec:
 
 If `Component`'s auto-generated image repository should be deleted after component deletion, add `image.redhat.com/delete-image-repo` annotation to the `Component`.
 
+### Update image repository token of the Component
+
+To refresh image repository access token, set annotation `image.redhat.com/generate: 'regenerate-token'` on the Component.
```

Comment:
also rest of readme should be checked, eg. now there will not be secret in `image.redhat.com/image` annotation